dotprogressbar/src/main/java/com/metagalactic/dotprogressbar/DotProgressBar.java

```diff
@@ -56,24 +56,24 @@ private void init(@Nullable AttributeSet attrs) {
             mDotColor = typedArray.getColor(R.styleable.DotProgressBar_dpb_dotColor, Color.GRAY);
             mDotRadius = typedArray.getDimension(R.styleable.DotProgressBar_dpb_dotRadius, 18);
             mGrowFactor = typedArray.getFloat(R.styleable.DotProgressBar_dpb_growFactor, 1.5f);
-            mDotSpacing = typedArray.getDimension(R.styleable.DotProgressBar_dpb_dotSpacing, 0);
+            mDotSpacing = typedArray.getDimension(R.styleable.DotProgressBar_dpb_dotSpacing, 8);
```

Comment:
This was undoing of a hack I put in last week so that dots won't overlap. It was set to 0 as larger radius was taken into consideration. Now we only look at actual dot size so need default so there is some space. We can now have overlapping dots without any clipping issues